src/tables.rs

```diff
@@ -1,3 +1,5 @@
+use strum::EnumCount;
+
```

Comment:
Please remove the extra line here and sort the imports.

```suggestion
use strum::EnumCount;
```